### ISSUE TITLE
Needed to add .data to createToken promise handler

### ISSUE THREAD

**Issue by dahannajr** [sguilly/loopback-nodejs-client#4]
I'm using LoopBack v3.19.0 with a custom extension to the User model.  Using the LNC, authentication against the custom endpoint works fine, but I needed to change

```javascript
this.token = result.id;

resolve(result.id);
```

to 

```javascript
this.token = result.data.id

resolve(result.data.id)
```

I'm not sure if this is a one-off because of the customer User model.

**Comment by sguilly** [comment]
In your custom route, you could add 'root: true' in return object

  User.remoteMethod('greet', {
          accepts: {arg: 'msg', type: 'string'},
          returns: {arg: 'greeting', type: 'string', root: true}
    });

see : https://loopback.io/doc/en/lb2/Remote-methods.html

**Comment by dahannajr** [comment]
It's not so much a custom route, as a custom Model which is inheriting from the User model.  So, in my NodeJS code, I'm using:

```javascript

var LoopbackClient = require('loopback-nodejs-client');
var loopbackClient = new LoopbackClient(url, email, password);
loopbackClient.setCustomLoginPath("/CustomerUser/login");

....

loobackClient
   .createToken()
   .then((token) => {
      ....
    }
    .catch((err) => {
      ....
    }
```

The call to createToken is working fine and doing what it should be doing.  But when I set DEBUG=LoopBackClient, I see that the debug entry from the LoopbackClient.js file on line 78 has the token ID stored in 

`result.data.id`

and not

`result.id`

**Comment by sguilly** [comment]
Fix in 1.5.5
